apps/finance/app/src/components/NewTransfer/Deposit.js

```diff
@@ -167,16 +168,20 @@ class Deposit extends React.Component {
       }
 
       const [tokenSymbol, tokenDecimals] = await Promise.all([
-        getTokenSymbol(api, address),
-        token.decimals().toPromise(),
+        getTokenSymbol(api, address).catch(() => {}),
```

Comment:
Super nitpicky but should we return explicit values (`null`) rather than implicit `undefined` in these cases?

Reply:
Yeah... we should probably settle on a convention for handling errors and their "safe" default values.

In the case of the balance check above, I opted for "0" since it'll show "no balances" but we could probably use "-1" as "couldn't find balance".

Reply:
I've now changed it to use `-1` for the balance if the balance call failed and default the symbol and name values to `''` (since they're optional strings for the token to implement).